src/file-1.xoo

```diff
@@ -0,0 +1,17 @@
+// dummy line 1
+// dummy line 2
+// violate HasTag rule: xoo on line 3
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:43683/organizations/org0/rules#rule_key=xoo%3AHasTag))

[See it in SonarCloud](http://127.0.0.1:43683/project/issues?id=sample&pullRequest=6144&issues=AWes4u8di6RaHGuv-QkR&open=AWes4u8di6RaHGuv-QkR)

[ISSUE_KEY]: AWes4u8di6RaHGuv-QkR